doc/nrf/libraries/bluetooth_services/mesh/lightness_srv.rst

```diff
@@ -103,6 +103,14 @@ This information is used to reestablish the correct Light level when the device
 If :kconfig:option:`CONFIG_BT_SETTINGS` is enabled, the Light Lightness Server stores all its states persistently using a configurable storage delay to stagger storing.
 See :kconfig:option:`CONFIG_BT_MESH_MODEL_SRV_STORE_TIMEOUT`.
 
+The Light Lightness Server can use the :ref:`emergency data storage (EMDS) <emds_readme>` together with persistent storage to:
+
+* Extend the flash memory life expectancy.
+* Reduce the use of resources by reducing the number of writes to flash memory.
+
+If option :kconfig:option:`CONFIG_EMDS` is enabled, the Light Lightness Server continues to store the default light and light range states to the flash memory through the settings library, but the last known non-zero Light level and whether the light is on or off are stored by using the :ref:`EMDS <emds_readme>` library.
```

Comment:
We should mention when `emds_load` shall be called (between `settings_load()` and `bt_mesh_start()`). Ideally, there should be a separate page in our docs explaining the sequence of calls during mesh initialization with and without settings, emds.

Actually, this is a generic section for all models that uses EMDS. Could it be moved, for example, to the Bluetooth mesh models overview page? http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/bluetooth_services/mesh/model_types.html

Reply:
There is a issue in our backlog to create a sample for this. Can we create a sample for this and that way show how this is done. If not we should have a section where we explain how the main file should be created, but that is more than just emds for models. So could we keep that outside this PR?

Reply:
Agree, lets create a separate task and keep it out of this PR